data_steward/tools/load_vocab.sh

```diff
@@ -12,12 +12,14 @@ tools/load_vocab.sh --app_id app_id --in_dir in_dir [--gcs_path gcs_path --datas
  --in_dir   in_dir   directory where vocabulary files are located
  --gcs_path gcs_path full GCS path to save transformed files
  --dataset  dataset  name of BigQuery dataset to create and load vocabulary
+ [--venv_dir venv_dir directory where virtual environment should be created]
 "
 SCRIPT_PATH="$(
```

Comment:
Since we're in this script, now is an opportune time to implement CH's previous suggestion and fix our `$BASE_DIR` and `$SCRIPT_PATH` setters so they are not reliant on which directory you are physically located in to execute.

```
BASE_DIR="$(git rev-parse --show-toplevel)"/data_steward
SCRIPT_PATH=${BASE_DIR}/tools
```

Then this should work whether you start if from `curation` or `curation/data_steward/tools`.